crates/router/tests/connectors/payu.rs

```diff
@@ -32,15 +32,29 @@ impl utils::Connector for Payu {
 
 #[actix_web::test]
 async fn should_authorize_card_payment() {
+    //Authorize Card Payment in PLN currenct
     let authorize_response = Payu {}
         .authorize_payment(
             Some(types::PaymentsAuthorizeData {
                 currency: enums::Currency::PLN,
+                browser_info: Some(BrowserInformation {
+                    user_agent: "".to_string(),
+                    accept_header: "".to_string(),
+                    language: "nl-NL".to_string(),
+                    color_depth: 24,
+                    screen_height: 723,
+                    screen_width: 1536,
+                    time_zone: 0,
+                    java_enabled: true,
+                    java_script_enabled: true,
+                    ip_address: Some("127.0.0.1".parse().unwrap()),
+                }),
```

Comment:
You could move this to a function which constructs `BrowserInformation` specific to this connector, instead of repeating this multiple times.